docs/arguments.md

```diff
@@ -1,5 +1,57 @@
 # Cortex Arguments Explained
 
+## Querier
+
+- `-querier.max-concurrent`
+
+   The maximum number of top-level PromQL queries that will execute at the same time, per querier process.
+
+- `-querier.query-parallelism`
+
+   This refers to database queries against the store (e.g. Bigtable or DynamoDB).  This is the max subqueries run in parallel per higher-level query.
+
+- `-querier.timeout`
+
+   The timeout for a top-level PromQL query.
+
+- `-querier.max-samples`
+
+   Maximum number of samples a single query can load into memory, to avoid blowing up on enormous queries.
```

Comment:
Where is this flag defined?   I don't recall it, or any limit like it in the querier.

Reply:
https://github.com/cortexproject/cortex/blob/ed1292b5b208e045e881d5f8507fc60ce9e3ee21/pkg/querier/querier.go#L43

It's a Prometheus limit really.